ttnn/cpp/pybind11/operations/core.hpp

```diff
@@ -282,6 +282,30 @@ void py_module(py::module& module) {
             py::arg("dtype") = std::nullopt,
             py::arg("memory_config") = std::nullopt,
             py::arg("device") = nullptr});
+
+    py::class_<DeviceComputeKernelConfig>(module, "DeviceComputeKernelConfig");
```

Comment:
have the class binding under a separate function py_module_types, above py_module

Reply:
Done